The flag added here avoids a warning on some version of AM.
Not much suitable for community commit because it is AM-version dependent.
--- a/Makefile.am	2017-12-19 18:05:56.000000000 +0000
+++ b/Makefile.am	2018-01-27 00:23:16.491144838 +0000
@@ -1,4 +1,5 @@
 ACLOCAL_AMFLAGS = -I m4
+AUTOMAKE_OPTIONS = subdir-objects
 
 pkgconfdir = $(libdir)/pkgconfig
 pkgconf_DATA = \
